src/lib/main.js

```diff
@@ -37,6 +38,26 @@ cm.Item({
   }
 });
 
+var readerModeKey = Hotkey({
+    combo: "accel-f1",
+    onPress: function() {
+        let url = tabs.activeTab.url;
+        let readerPrefix = 'about:reader?url=';
+        let matchLoc = url.search(readerPrefix.substr(0,12));
```

Comment:
To me it is better to rewrite this line like: `let matchLoc = url.indexOf(readerPrefix) === 0`